src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/custom.py

```diff
@@ -16,8 +16,8 @@
 
 logger = get_logger(__name__)
 
-_CLOUD_CONSOLE_WARNING_TEMPLATE = ("Azure Cloud Shell automatically authenticates the user account it was initially"
-                                   " launched under, as a result 'az %s' is disabled.")
+_CLOUD_CONSOLE_WARNING = ("Logout successful. Re-login to your initial Cloud Shell identity with 'az login --msi'."
```

Comment:
Are we still using the `--msi` flag then?

Reply:
we do plan to use `--msi`, but we are still in the discussion for a better and still short name. I should address that in  #5744